extensions/wikia/ArticleComments/classes/ArticleCommentList.class.php

```diff
@@ -1080,4 +1080,28 @@ static public function undeleteComplete($oTitle, $oUser, $reason) {
 		wfProfileOut( __METHOD__ );
 		return true;
 	}
+
+	/**
+	 * Enables commenting for an article if it is moved to a namespace that allows it
+	 * and disables it if it does not.
+	 * @param object Title $oOldTitle An object for the old article's name
+	 * @param object Title $oNewTitle An object for the new article's name
+	 * @param object User $oUser
+	 * @param integer $iOldId
+	 * @param integer $iNewId
+	 * @return bool
+	 */
+	public static function onTitleMoveComplete( Title &$oOldTitle, Title &$oNewTitle, User &$oUser, $iOldId, $iNewId ) {
+		global $wgArticleCommentsNamespaces;
+		wfProfileIn( __METHOD__ );
+
+		if ( ArticleComment::isBlog( $oNewTitle ) || in_array( $oNewTitle->getNamespace(), $wgArticleCommentsNamespaces ) ) {
```

Comment:
What about if a blog was renamed, but had comments disabled intentionally (i.e. to correct a title typo)? I think the property should probably survive the move if it was set explicitly on an existing blog (so if the old title was also a blog for example).